examples/X11/X11.cpp

```diff
@@ -47,8 +47,7 @@
     // Setup a perspective projection
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
-    static const float pi = 3.141592654f;
-    float extent = std::tan(90.0f * pi / 360.0f);
+    float extent = std::tan(sf::degrees(90).asRadians());
```

Comment:
I think this should be `float extent = std::tan(sf::degrees(45).asRadians());` instead.

Reply:
Indeed, 90/360 * pi = pi/4 rad = 180°/4 deg

Reply:
Thank you! It should be fixed now.